.github/workflows/test.yaml

```diff
@@ -7,6 +7,7 @@ on:
       - patch-dev
   pull_request:
 env:
+  CI: 1
```

Comment:
Not necessary in theory as GH Actions should be picked up automatically by the library we are using.